front-end/src/styles/DashboardFooter.css

```diff
@@ -0,0 +1,14 @@
+.bg-color {
+    background-color: rgb(240, 226, 219);
+}
```

Comment:
Maybe somewhere here we can implement an outline to help the objects in the screen pop out more